expression-language/6.0/_index.md

```diff
@@ -18,29 +18,26 @@ This release makes the dependency on the java.desktop module optional, removes r
 * All references to the Java SecurityManager and associated APIs have been removed.
 
 ### Minimum Java SE Version
-**Java SE 11 or higher**
+**Java SE 17 or higher**
 
 # Details
 
 * [Jakarta Expression Language 6.0 Release Record](https://projects.eclipse.org/projects/ee4j.el/releases/6.0.0)
-
-<!--
-
-The following can be uncommented and version information updated as they become available.
-
-    * [Jakarta EE Platform 10 Release Plan](https://jakartaee.github.io/platform/jakartaee10/JakartaEE10ReleasePlan)
-* [Jakarta Expression Language 5.0 Specification Document](./jakarta-expression-language-spec-5.0.pdf) (PDF)
-* [Jakarta Expression Language 5.0 Specification Document](./jakarta-expression-language-spec-5.0.html) (HTML)
-* [Jakarta Expression Language 5.0 Javadoc](./apidocs)
-* [Jakarta Expression Language 5.0 TCK](https://download.eclipse.org/jakartaee/expression-language/5.0/jakarta-expression-language-tck-5.0.0.zip)  ([sig](https://download.eclipse.org/jakartaee/expression-language/5.0/jakarta-expression-language-tck-5.0.0.zip.sig),  [sha](https://download.eclipse.org/jakartaee/expression-language/5.0/jakarta-expression-language-tck-5.0.0.zip.sha256),  [pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
+    * [Jakarta EE Platform 11 Release Plan](https://jakartaee.github.io/platform/jakartaee11/JakartaEE11ReleasePlan)
+* [Jakarta Expression Language 6.0 Specification Document](./jakarta-expression-language-spec-6.0.pdf) (PDF)
+* [Jakarta Expression Language 6.0 Specification Document](./jakarta-expression-language-spec-6.0.html) (HTML)
+* [Jakarta Expression Language 6.0 Javadoc](./apidocs)
+* [Jakarta Expression Language 6.0 TCK](https://download.eclipse.org/jakartaee/expression-language/6.0/jakarta-expression-language-tck-6.0.0.zip)  ([sig](https://download.eclipse.org/jakartaee/expression-language/6.0/jakarta-expression-language-tck-6.0.0.zip.sig),  [sha](https://download.eclipse.org/jakartaee/expression-language/6.0/jakarta-expression-language-tck-6.0.0.zip.sha256),  [pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
 * Maven coordinates
-  * [jakarta.el:jakarta.el-api:jar:5.0.0](https://search.maven.org/artifact/jakarta.el/jakarta.el-api/5.0.0/jar)
+  * [jakarta.el:jakarta.el-api:jar:6.0.0](https://search.maven.org/artifact/jakarta.el/jakarta.el-api/6.0.0/jar)
 -->
 
 
 # Compatible Implementations
 
-* Eclipse ExpressLy (TBC)
+* Tomcat 11.0.0-M18-SNAPSHOT (Java 17)
+* Tomcat 11.0.0-M18-SNAPSHOT (Java 21)
+* ExpressLY (TBD)
```

Comment:
If we're doing a release I'd drop this for now.
We can add it back in when it's officially a compatible implementation
```suggestion
```